docs/tutorials/03_vqe_simulation_with_noise.ipynb

```diff
@@ -0,0 +1,452 @@
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import pylab\n",
+    "\n",
+    "pylab.rcParams[\"figure.figsize\"] = (12, 4)\n",
+    "pylab.plot(counts, values)\n",
+    "pylab.xlabel(\"Eval count\")\n",
+    "pylab.ylabel(\"Energy\")\n",
+    "pylab.title(\"Convergence with no noise\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Performance *with* noise\n",
+    "\n",
+    "Now, let's add noise to our simulation. In particular, you will extract a noise model from a (fake) device. As stated in the introduction, it is also possible to create custom noise models from scratch, but this task is beyond the scope of this notebook.\n",
+    "\n",
+    "First, you need to get an actual device backend and from its `configuration` and `properties` you can setup a coupling map and a noise model to match the device. Note: You can also use this coupling map as the entanglement map for the variational form if you choose to."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "NoiseModel:\n",
+      "  Basis gates: ['cx', 'id', 'rz', 'sx', 'x']\n",
+      "  Instructions with noise: ['id', 'sx', 'x', 'measure', 'cx']\n",
+      "  Qubits with noise: [0, 1, 2, 3, 4]\n",
+      "  Specific qubit errors: [('id', (0,)), ('id', (1,)), ('id', (2,)), ('id', (3,)), ('id', (4,)), ('sx', (0,)), ('sx', (1,)), ('sx', (2,)), ('sx', (3,)), ('sx', (4,)), ('x', (0,)), ('x', (1,)), ('x', (2,)), ('x', (3,)), ('x', (4,)), ('cx', (3, 4)), ('cx', (4, 3)), ('cx', (3, 1)), ('cx', (1, 3)), ('cx', (1, 2)), ('cx', (2, 1)), ('cx', (0, 1)), ('cx', (1, 0)), ('measure', (0,)), ('measure', (1,)), ('measure', (2,)), ('measure', (3,)), ('measure', (4,))]\n"
+     ]
+    }
+   ],
+   "source": [
+    "from qiskit_aer.noise import NoiseModel\n",
+    "from qiskit.providers.fake_provider import FakeVigo\n",
+    "\n",
+    "# fake providers contain data from real IBM Quantum devices stored in Qiskit Terra,\n",
+    "# and are useful for extracting realistic noise models.\n",
```

Comment:
real IBM Quantum devices stored in Qiskit ~Terra~
  
 Think we should drop the Terra part here in this comment and leave it just as Qiskit